src/common/OptionButton.js

```diff
@@ -35,13 +33,17 @@ export default class OptionButton extends PureComponent<Props> {
   props: Props;
 
   render() {
-    const { label, onPress } = this.props;
+    const { label, onPress, Icon } = this.props;
+    const { icon, rightItem, settingsIcon } = this.context.styles;
```

Comment:
Get `styles` from destructing.
`const { styles } = this.context;`
In the whole project we are following same code styles :)

Reply:
Understood. 👍